type-reducer/experimental_rename_only_mapped_names.txt

```diff
@@ -0,0 +1,14 @@
+### Rename only
+GRPCRouteRules->GRPCRouteRule
+HTTPRouteRules->HTTPRouteRule
+HTTPRouteRulesFilters->HTTPRouteFilter
+GRPCRouteRulesMatches->GRPCRouteMatch
+HTTPRouteRulesMatches->RouteMatch
+HTTPRouteRulesTimeouts->HTTPRouteTimeout
+GRPCRouteRulesBackendRefs->GRPCBackendReference
+HTTPRouteRulesBackendRefs->HTTPBackendReference
+GRPCRouteRulesMatchesMethod->GRPCMethodMatch
+HTTPRouteRulesMatchesMethod->HTTPMethodMatch
+HTTPRouteRulesBackendRefsFilters->HTTPRouteBackendFilter
+HTTPRouteRequestRedirect->RequestRedirect
+HTTPRouteRulesMatchesPath->PathMatch
```

Comment:
Is it potentially better to not check in these mapping files because they will be regenerated on every generator run anyway? Or am I misunderstanding?